api/app/settings/common.py

```diff
@@ -53,6 +53,9 @@
 
 HOSTED_SEATS_LIMIT = env.int("HOSTED_SEATS_LIMIT", default=0)
 
+MAX_SEATS_IN_FREE_PLAN = 1
+MAX_PROJECTS_IN_FREE_PLAN = 1
+
```

Comment:
@matthewelwell They are not settable from env, but do we want to expose these constants as settings?

Reply:
I don't see a reason to move them from `organisations.subscriptions.constants` really, no.

Reply:
@matthewelwell @khvn26 please refer the PS section in the PR description above on why I made this change. 

Apparently, there are a few tests that create multiple projects for a single organisation on the default free subscription and error out. 

I can maybe look into changing the default subscription for tests to something else.

Reply:
Good point! I don't love the fact that this will cause the tests to fail when not using `DJANGO_SETTINGS_MODULE=app.settings.test` but I don't necessarily see another option here. We might just see some pain while we ensure that everything in our CI process is using that module.

Reply:
https://github.com/Flagsmith/flagsmith/blob/main/api/pyproject.toml#L36C1-L38C1 - I see pytest to be defaulted to `app.settings.test`, what else can break here?

Reply:
Hopefully nothing! 😄